benchmarks/DiscoveryBenchmark.php

```
@@ -130,12 +122,9 @@ public function contextualize(): void {
         }
 
       };
-
-      Environment::addContext($context);
     }
 
-    // Access contexts to ensure they are loaded.
-    Environment::context();
+    Environment::init(contexts: $contexts);
```

PHPMD (2.15.0)

Avoid using static access to class '\DrevOps\EnvironmentDetector\Environment' in method 'benchAddContext'. (undefined)

(StaticAccess)